shell/platform/fuchsia/dart-pkg/zircon/lib/src/system.dart

```diff
@@ -88,7 +95,7 @@ class WriteResult extends _Result {
 
 @pragma('vm:entry-point')
 class GetSizeResult extends _Result {
-  final int size;
+  final int? size;
```

Comment:
This migration to null safety looks good.

However, if it is idiomatic to check status before using other fields of the result classes, then we might also consider introducing getter which would return non-nullable value, as it is much easier to work with non-nullable values. Something along the lines:

```dart
class GetSizeResult extends _Result {
  final int? _size;
  int get size => _size!;

  @pragma('vm:entry-point')
  const GetSizeResult(final int status, [this._size]) : super(status);

  @override
  String toString() => 'GetSizeResult(status=$status, size=$_size)';
}
```

This can be a breaking change, so it might make sense to do this in a separate PR.

Reply:
That is actually a really good point. Right now folks would have to check status or else they would crash when they tried to check the size. This would probably be the correct time to migrate this.

Reply:
Updated.